src/devices/Bmp180/samples/Bmp180.Sample.cs

```diff
@@ -30,23 +31,27 @@ static void Main(string[] args)
 
                 //read values
                 Temperature tempValue = i2cBmp280.ReadTemperature();
-                Console.WriteLine($"Temperature {tempValue.Celsius} \u00B0C");
+                Console.WriteLine($"Temperature: {tempValue.Celsius:0.#}\u00B0C");
                 var preValue = i2cBmp280.ReadPressure();
-                Console.WriteLine($"Pressure {preValue.Hectopascal} hPa");
-                double altValue = i2cBmp280.ReadAltitude();
-                Console.WriteLine($"Altitude {altValue:0.##} m");
+                Console.WriteLine($"Pressure: {preValue.Hectopascal:0.##}hPa");
+
+                // Note that if you already have the pressure value and the temperature, you could also calculate altitude by 
+                // calling WeatherHelper.CalculateAltitude(preValue, Pressure.MeanSeaLevel, tempValue) which would be more performant.
+                var altValue = i2cBmp280.ReadAltitude(Pressure.MeanSeaLevel);
+
+                Console.WriteLine($"Altitude: {altValue:0.##}m");
                 Thread.Sleep(1000);
 
                 //set higher sampling
                 i2cBmp280.SetSampling(Sampling.UltraLowPower);
 
                 //read values
                 tempValue = i2cBmp280.ReadTemperature();
-                Console.WriteLine($"Temperature {tempValue.Celsius} \u00B0C");
+                Console.WriteLine($"Temperature: {tempValue.Celsius:0.#}\u00B0C");
                 preValue = i2cBmp280.ReadPressure();
-                Console.WriteLine($"Pressure {preValue.Hectopascal} hPa");
-                altValue = i2cBmp280.ReadAltitude();
-                Console.WriteLine($"Altitude {altValue:0.##} m");
+                Console.WriteLine($"Pressure: {preValue.Hectopascal:0.##}hPa");
+                altValue = WeatherHelper.CalculateAltitude(preValue, Pressure.MeanSeaLevel, tempValue);

```

Comment:
NIT: seems like you forgot to change this one back to use i2cBmp280.ReadAltitude()

Reply:
Done (pending commit)